va/va.c

```diff
@@ -1225,7 +1225,7 @@ VAStatus vaMFSubmit (
 )
 {
     VADriverContextP ctx;
-    VAStatus vaStatus;
+    VAStatus vaStatus = VA_STATUS_SUCCESS;
 
     CHECK_DISPLAY(dpy);
     ctx = CTX(dpy);
```

Comment:
It is not needed if returning VA_STATUS_ERROR_UNIMPLEMENTED at once when ctx->vtable->vaCreateMFContext is NULL, 

BTW Could you please to remove all trailing whitespaces in this patch, then squash the two commits into one commit?

Reply:
Done